decidim-conferences/app/serializers/decidim/conferences/download_your_data_conference_invite_serializer.rb

```diff
@@ -10,15 +10,14 @@ def serialize
           sent_at: resource.sent_at,
           accepted_at: resource.accepted_at,
           rejected_at: resource.rejected_at,
-          user: {
```

Comment:
I guess this doesn't make sense to be exported is the person that exports this is actual user (ex: current_user).

Reply:
Exactly, that was my reasoning: we're adding the user metadata but the user already know who they are 😄